src/test/java/LottoShopTest.java

```diff
@@ -0,0 +1,45 @@
+import domain.Lotto;
+import domain.LottoShop;
+import domain.Lottos;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LottoShopTest {
```

Comment:
```suggestion
@SuppressWarnings("NonAsciiCharacters") // 한글과 같은 아스키 코드가 아닌 문자에 밑줄이 쳐지지 않음
@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class) // UnderScore(_)가 공백으로 표시
public class LottoShopTest {
```
P3) 한글명으로 테스트 이름을 작성하면 인텔리제이에 밑줄이 보이기도 하고, 테스트 실행하면 언더바가 보이는데 이걸 없애주는 방법이에요

Reply:
P2) given - when - then 패턴 찾아보시면 좋아보여요. 아마 지금 테스트는 상희님이 잘 구분지어 두어서 주석만 추가하면 끝날 것 같네요 ([참고 글](https://github.com/meshkorea/front-end-engineering/blob/main/tests/given_when_then.md) - 프론트엔드 문서를 가지고 왔는데, 여기가 딱 필요한 내용만 적어두어서 가지고 왔어요)

Reply:
P3) 테스트 코드 파일 만드실 때 직접적으로 클래스 파일을 추가한거죠?? 이거는 인텔리제이에서 Cmd + Shift + T를 누르면 main 안에 세부 패키지랑 똑같은 구조로 클래스 파일을 만들어줘서 참고하면 좋아보여요
ex. `Lotto.java`에서 Cmd + Shift + T 입력 > Create New Test (엔터키) > OK (엔터키)